packages/webview_flutter/lib/webview_flutter_tizen.dart

```diff
@@ -36,6 +39,219 @@ enum _PlatformViewState {
   ready,
 }
 
+bool _factoryTypesSetEquals<T>(Set<Factory<T>>? a, Set<Factory<T>>? b) {
+  if (a == b) {
+    return true;
+  }
+  if (a == null || b == null) {
+    return false;
+  }
+  return setEquals(_factoriesTypeSet(a), _factoriesTypeSet(b));
+}
+
+Set<Type> _factoriesTypeSet<T>(Set<Factory<T>> factories) {
+  return factories.map<Type>((Factory<T> factory) => factory.type).toSet();
+}
+
+/// Builds an Tizen webview.
+///
+/// This is used as the default implementation for [WebView.platform] on Tizen. It uses a method channel to
+/// communicate with the platform code.
+class TizenWebView implements WebViewPlatform {
+  static void register() {
+    WebView.platform = TizenWebView();
+  }
+
+  @override
+  Widget build({
+    required BuildContext context,
+    required CreationParams creationParams,
+    WebViewPlatformCreatedCallback? onWebViewPlatformCreated,
+    Set<Factory<OneSequenceGestureRecognizer>>? gestureRecognizers,
+    required WebViewPlatformCallbacksHandler webViewPlatformCallbacksHandler,
+  }) {
+    assert(webViewPlatformCallbacksHandler != null);
```

Comment:
vscode says this is always true because it can't be null.

Reply:
Most codes come from the original code.(include this part!)

Reply:
This is not just a copy. You're defining a new class with inheritance like below, so you don't have to keep the exact same thing.

Does this make it difficult to sync with upstream code future?

```dart
class TizenWebView implements WebViewPlatform
```

Reply:
In fact, All I need is a very small difference branch and I want most of the logic and code to be identical to that of original.
Of course, the reason for doing this is to apply upstream easily.

I understand your intentions, but  I don't make any contribution to platform view logic and I want to make this clear & reveal it in code for now. (This is my intention..if this is strange, I am willing to change it at any time!)